library/Rules/Cnpj.php

```diff
@@ -19,7 +19,7 @@ public function validate($input)
         $c = preg_replace('/\D/', '', $input);
         $b = [6, 5, 4, 3, 2, 9, 8, 7, 6, 5, 4, 3, 2];
 
-        if (strlen($c) != 14) {
+        if (strlen($c) != 14 || preg_match("/^{$c[0]}{14}$/", $c)) {
```

Comment:
@henriquemoody He's trying to avoid repeated strings of only one digit like `11111111111111`, so `/^1{14}$/` is correct, btw, implementation sucks IMHO.